src/features/import/utils/problems/predictProblems.ts

```diff
@@ -111,7 +111,8 @@ export function predictProblems(
             ) {
               accumulateFieldProblem(column.field, rowIndex);
             } else if (column.field == 'phone' || column.field == 'alt_phone') {
-              if (!isValidPhoneNumber(value.toString(), country)) {
+              const phoneValue = value.toString().replaceAll(/[^+\d]/g, '');
```

Comment:
This regex finds (and through `replaceAll()` removes) all characters except numbers and the literal `+`. I think it's a nice solution!